standards/commits.md

```diff
@@ -1,52 +1,81 @@
+- **_chore_**: Updating gulp, webpack, package.json files. This is developer-facing _only_.
+- **_workaround_**: Temporary fix until a more robust solution is found or until other factors are resolved.
+
+`scope`: Narrow the scope of the commit to a one or two word description in parentheses
+
+`subject`: Favor imperative mood, present tense, active voice, and start with verbs. Don't use a period at the end. Think of it as a newspaper headline.
+
+`body` (optional): If necessary, provide additional context that can help other developers in the future. This is normally unnecessary but some use cases are:
+
+- If the commit contains a new package you've added to the project
+- If the commit contains a change to your build that you need to notate
+- If the commit includes changes that would benefit from an explanation and from additional context.
+- If the commit is the last in a series that will become a Pull Request and you want to communicate something to your senior developer.
+
+## Examples
+
+Say you have changed four files:
+
+> app.component.ts
+> app.router.ts
+> login.component.ts
+> login.component.scss
+
+There are at least two _scopes_ being dealt with here: the **app.component** scope and the **login.component** scope.
+
+- `git add` the **app.component.ts** and the **app.router.ts** and create a commit for those files.
+
+  > `git commit -m "refactor(app-component): import user service and add routes"`
+
+  _Now you have 1 commit, dedicated to the **app.component** scope._
+
+- Next, `git add` the login files and create a new commit message for these files.
+  > `git commit -m "feat(login): create/setup"`
+        _Note that this message is less verbose because it isnt necessary to add detail_
+
+## Automation Tools 🛠
+
+There are many tools available to help you ***enforce*** the commit convention rules via git hooks. This way you can always be sure that your commit will satisfy the rules before they're added to the history.
+
+One example, if you're using npm/package.json in your project, you can install [commitlint](https://github.com/conventional-changelog/commitlint) package. This adds a commit message linter to your command line. There are many plugins available for it with pre-configured convention rules, you can check them out [here](https://github.com/conventional-changelog/commitlint#shared-configuration). If those don't meet your needs you can extend them using a commitlint [configuration file](https://github.com/conventional-changelog/commitlint#config). You can then add the commitlint as a hook using Husky ([directions here](https://github.com/conventional-changelog/commitlint#getting-started)). After you set that up every time you try to run `git commit` it will first pass your commit message through the linter and either it will succeed or get rejected with a violation reason if it doesn't comply with the rules.
```

Comment:
I feel like a bulleted list might be a better way to visualize some of these tools and sets of tools. Do we have any examples within projects for these kinds of git commit tool setups?